velox/docs/functions/spark/bitwise.rst

```diff
@@ -0,0 +1,24 @@
+=================
+Bitwise Functions
+=================
+
+.. js:function:: bitwise_and(x, y) -> [same as input]
```

Comment:
Can you explain the usage of js::function in the PR description?

Reply:
Ok, In fact, If we use function::, the default domain is py::function. So I need to distinguish between spark function and presto function. So I think the most straightforward way is to use different namespace like js. 
Here is a official explanation. https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#directive-default-domain. 
![image](https://user-images.githubusercontent.com/14065538/213603646-5257b26d-41d1-433b-98a8-abf5a5a75727.png)